### PR TITLE
SelectBox/TagBox: Update notes and code snippets (T1156423)

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The example below shows how to define the **fieldTemplate**. If you want to cust
             fieldTemplate(data, container) {
                 container.append(
                     $('<div>').dxTextBox({
-                        text: data.Category + ": " + data.Name,
+                        value: data && data.Name,
```
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]: Why did you change the field's text?

Comment (sergepilipchuk) [reply api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]: Please, refer to the T1156423 ticket

Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md]: It says nothing about why you ditched data.Category

**Review by vladaskorohodova — APPROVED** [review]
